gradle.properties

```diff
@@ -10,7 +10,7 @@ cordaProductVersion = 5.0.0-INTEROP
 ## IMPORTANT: 
 ## The interop feature branches track api revisions separately to the mainline branch.
 ## API version of last merge from corda mainline: 731
-cordaApiRevision = 2
+cordaApiRevision = 3
```

Comment:
Need to bump up the version to 5, I just increased to to 4 for synching up with release branch.